module-2/backend/week_one.md

```diff
@@ -5,30 +5,114 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+The five common verbs: 
+  ** 1. Get ** Get's purpose is to receive information.
+  ** 2. Post ** Post's purpose is to send information.
+  ** 3. Put ** Put's purpose is to update information.
+  ** 4. Delete ** Delete's purpose is to delete information.
+  ** 5. Head ** Head returns just the header, which I imagine is useful for something I haven't encountered yet.
 2. What is Sinatra?
+  * Sinatra is a DSL alternative to Rails that allows me to run a web app. It uses Rack to help communicate with the server.*
 4. What is MVC?
+  *MVC is: Model View Controller. This is the architecture for web apps.*

```

Comment:
An architecture, for sure. Not the only approach. Important to also be familiar with what each piece does, but we will certainly be discussing more in the weeks to come.